.gitlab-ci.yml

```diff
@@ -413,3 +415,18 @@ check-squash-into-staging:
     - eval $(ssh-agent -s)
   script:
     - node scripts/staging-ci/check-squash-into-staging.js
+
+########################################################################################################################
+# Bump e2e chrome version
+########################################################################################################################
+
+bump-chrome-version-scheduled:
```

Comment:
should we be notified when it succeed or failed?

Reply:
Yes sure